syntax = "proto3";

message SensorConfiguration {
  string name = 1;
  int32 initLatitude = 2;
  int32 initLongitude = 3;
  enum DeviceType {
  	GNSS0 = 0;
  	GNSS1 = 1;
  }
  DeviceType type = 5;
}
